### ISSUE TITLE
[TEST][FLAKY] test_quantize_pass

### ISSUE THREAD

**Issue by tqchen** [dmlc/tvm#2718]
http://ci.tvm.ai:8080/blue/rest/organizations/jenkins/pipelines/tvm/branches/PR-2546/runs/18/nodes/36/log/?start=0

cc @ZihengJiang , please fix help this

**Comment by jroesch** [comment]
This is also blocking the #2546 PR.

**Comment by ZihengJiang** [comment]
I will try to fix this. Let's move the test case for now

**Comment by zhiics** [comment]
FYI, I checked out the docker image and narrowed down to the following 3 files:
```bash
TVM_FFI=cython python -m nose -v tests/python/relay/test_ir_text_printer.py tests/python/relay/test_op_level5.py tests/python/relay/test_pass_quantize.py
```

**Comment by icemelon9** [comment]
http://ci.tvm.ai:8080/blue/rest/organizations/jenkins/pipelines/tvm/branches/PR-3362/runs/1/nodes/36/log/?start=0

This flaky test happens again

**Comment by icemelon9** [comment]
Fixed in #3362
